networking/dht-router/serialization/cl_dht_addr_announcement.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.14.0
+// 	protoc-gen-go v1.24.0
+// 	protoc        v3.12.3
```

Comment:
Should these version numbers go backwards?

Reply:
In principle, no. But since the generated code is identical, it's fine.